app/controllers/prospects_controller.rb

```diff
@@ -0,0 +1,65 @@
+class ProspectsController < ApplicationController
+  skip_before_action :authenticate_user!
```

Comment:
First things first: Any user should be able to create an account in cdx.
You are seeing cdx as a closed product where no one can create an account without invitation or proper authorization, but for most of the cases that's not true.

Having said that, there is an ongoing development related to this: https://github.com/instedd/cdx/issues/371 and https://github.com/instedd/cdx/tree/feature/371-single_tenant_deployment

The idea behind this is that in a single tenant deploy the user would be able to create an account but he won't be able to do anything else. Then an administrator would then give him access through a policy and from there he'll be able to access the application.

The "request access" feature would then be a part of the new institution screen, with an autocomplete in the name, the user will be able to send a request to the admin of that institution. If there is no institution left to send requests, an empty page will be displayed saying that his request is waiting for approval.

The invite screen will be a part of the create policy screen, from there you can choose an existing user to assign the policy. The idea is that besides choosing an existing user, you have a text box where you can type an email and the invite to that user will be sent.